src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
```

Comment:
Suggest importing these from the components package.

```suggestion
import { Icon, Link } from '@terra-ui-packages/components';
```

The `icon` render function has been replaced with an `Icon` component, so the code using it will have to be adapted slightly.